### PR TITLE
Added Docker support

### PR DESCRIPTION
**Pull request by jermnelson**
Allows BFE to run as a Docker container, includes Dockerfile and README directions

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM node:8
+
+ENV HOME /opt/bfe
+ENV REPO https://github.com/sul-dlss/bfe.git
```
Comment (jgreben) [inline docker/Dockerfile]: Should this be `https://github.com/ld4p/bfe.git` ?

File: docker/bfe-config.js [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+        var config = {
+/*            "logging": {
+                "level": "DEBUG",
+                "toConsole": true
+            },*/
+            "url" : "http://editor:8000",
+            "baseURI": "http://ld4p.org/",
+            "basedbURI": "http://ldp4p.org:8000",
+            "resourceURI": "http://ldp4p.org/resources",
+            "profiles": [
```
Comment (jgreben) [inline docker/bfe-config.js]: If the repo that docker clones is going to be `https://github.com/sul-dlss/bfe.git` then we may need to do something about the profiles that are loaded: trim this array, and also add the default profiles we will use to `static/profiles/bibframe` (which does not currently exist in this repo).

File: docker/bfe-config.js [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+			    "static/profiles/bibframe/BIBFRAME 2.0 DDC.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Item.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Identifiers.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Note.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Rare Materials.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Sound Recording: Audio CD-R.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 Load.json",
+                "static/profiles/bibframe/BIBFRAME 2.0 IBC.json"
+   		    ],
+            "startingPoints": [
```
Comment (jgreben) [inline docker/bfe-config.js]: We'd also need to edit/simplify the startingPoints menu items, if the above assumptions are true.

**Review by jgreben — CHANGES_REQUESTED** [review]
@jermnelson I added my comments. Let me know what you think.

File: docker/bfe-config.js (hunk 2 above)
Comment (jermnelson) [reply docker/bfe-config.js]: Agree, for now how does having one Audio Recording profile and one Video (maybe the Moving Image BlueRay DVD) profiles. Any suggestions on other profiles we should keep would be great.
  
I think the default profiles that were in `static/profiles/bibframe` are now being loaded from the loc website with a GET call i.e. http://id.loc.gov/ontologies/bibframe/Instance.json etc.

**Comment by jermnelson** [comment]
Having issues trying to run recto/verso/bfe in a single docker container or in a docker-compose multiple docker images. For now I'll close this request but circle back when we profiles loading through the bfe instead of externally through an external verso instance. I don't think the editor needs this type of external service.

**Pull request closed without merging**